components/Header.jsx

```diff
@@ -1,11 +1,18 @@
 import AccountInfo from "@/components/AccountInfo";
 import styles from "@/styles/components/header.module.scss";
+import Link from "next/link";
+import { useRouter } from "next/router";
 
 export default function Header() {
+	const router = useRouter();
+	const homeLink =
```

Comment:
1. It would make more sense to return `null` rather `undefined `as one represemts empty value and the other represents value that wasn't defined. loose terms but still.
2. Would be cleaner to conditionaly render the element on the return value itself using the Logical AND operator

Reply:
noted